CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+# Contributing to the Differential Datalog project
+
+We appreciate your interest in contributing to DDlog.  Please read our
+[Developer Certificate of Origin](https://cla.vmware.com/dco). All
+contributions to this repository must be signed as described on that
+page. Your signature certifies that you wrote the patch or have the
+right to pass it on as an open-source patch.
+
+## Project management
+
+We mostly use github tools for managing the project.  Bugs and
+questions should be filed as [github
+issues](https://github.com/vmware/differential-datalog/issues).
+Improvements should be submitted as [pull
+requests](https://github.com/vmware/differential-datalog/pulls).
+
+## Contributor License Agreement
+
+You will need to sign a CLA (Contributor License Agreement) to
+contribute code to DDlog under an MIT license.  This is standard.
+
+## Using git to contribute.
+
+Fork the repository using the "fork" button on github, by following these instructions:
+https://help.github.com/articles/fork-a-repo/.  The main step is the following:
+
+```shell
+git remote add upstream https://github.com/vmware/database-stream-processor.git
```

Comment:
wrong repo

Reply:
Copied this from the link you sent. But it is wrong...